woocommerce/woocommerce-android
# Issue/6132 order creation long customer note

Closes #6132 - this PR adds `WCReadMoreTextView` which automatically shows a "Read more" button after a specified number of lines, and this new component is used for the customer note during order creation.

To test:

- Add a large customer note during order creation
- Verify that a "Read more" button appears to display the entire note
- Replace the long note with a short one
- Verify the button no longer appears

![long](https://user-images.githubusercontent.com/3903757/161336870-93623128-4ed4-4bf8-b7e4-739349597482.png)


![short](https://user-images.githubusercontent.com/3903757/161337136-7dc6f01d-e942-4961-b22e-c175d9dc2b6b.png)
